docs/topics/wasm/wasm-publish-artifacts.md

```diff
@@ -0,0 +1,57 @@
+[//]: # (title: Publish artifacts)
+
+Kotlin/Wasm can generate artifacts to publish as a site on static hosting services. This tutorial
+demonstrates how to generate artifacts for a [Compose Multiplatform](https://www.jetbrains.com/lp/compose-multiplatform/)
+ app and publish them as a site on [GitHub pages](https://pages.github.com/).
+
+## Before you start
+
+1. Download and install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/).
+2. Clone the [Kotlin/Wasm examples](https://github.com/Kotlin/kotlin-wasm-examples/tree/main) repository
+   by selecting **File** | **New** | **Project from Version Control** in IntelliJ IDEA.
+
+   You can also clone it from the command line:
+
+   ```bash
+   git clone git@github.com:Kotlin/kotlin-wasm-examples.git
+   ```
+
+## Generate artifacts
+
+1. Open the **Gradle** tool window: **View** | **Tool Windows** | **Gradle**.
```

Comment:
Maybe this whole thing would be easer if we opened compose-example directory as a project in IDE instead of the root repo, but I don't know if this can be done in IDE 🤷🏻 . Separate project repo would make it ideal, of course :(

Reply:
As discussed on Slack, I added an additional step to the instructions to load Gradle scripts when the notification appears. Then the following step works 👍